app/src/main/java/io/apicurio/registry/rest/v2/AbstractResourceImpl.java

```diff
@@ -42,7 +42,7 @@ public abstract class AbstractResourceImpl {
     @Context
     HttpServletRequest request;
 
-    @ConfigProperty(name = "registry.apis.v2.base-href", defaultValue = "_")
+    @ConfigProperty(name = "apicurio.apis.v2.base-href", defaultValue = "_")
```

Comment:
`*.uri` suffix?

Reply:
In this case I disagree, it's not technically a unit, and it's stated in the description.